scwx-qt/scwx-qt.cmake

```diff
@@ -504,6 +505,12 @@ add_custom_target(scwx-qt_generate_versions ALL
 
 add_dependencies(scwx-qt scwx-qt_generate_versions)
 
+add_custom_target(scwx-qt_update_radar_sites ALL
```

Comment:
The CMake target works well from Visual Studio. The only suggestion I have here is maybe removing `ALL`, as we probably want to run the target manually/selectively.

```
Build started at 12:08 AM...
1>------ Build started: Project: scwx-qt_update_radar_sites, Configuration: Debug x64 ------
1>Reading Current Sites from 'C:/Users/dpaul/git/supercell-wx/scwx-qt/res/config/radar_sites.json'
1>Getting NEXRAD stations
1>Getting TDWR stations
1>Processing Data
1>CUSTOMBUILD : warning : Station 'KHDC' not found in noaa data
1>CUSTOMBUILD : warning : Station 'KLIX' not found in noaa data
1>Saving Updated Sites to 'C:/Users/dpaul/git/supercell-wx/scwx-qt/res/config/radar_sites.json'
1>[93mWARNING: Updating radar sites may break tests in 'test/source/scwx/qt/config/radar_sites.test.cpp'[39m
1>Building Custom Rule C:/Users/dpaul/git/supercell-wx/scwx-qt/CMakeLists.txt
1>Done building project "scwx-qt_update_radar_sites.vcxproj".
========== Build: 1 succeeded, 0 failed, 1 up-to-date, 0 skipped ==========
========== Build completed at 12:08 AM and took 10.953 seconds ==========
```